src/layers/Dialog/DialogTheme.tsx

```diff
@@ -10,36 +10,20 @@ export interface DialogTheme {
   };
 }
 
-const baseTheme: DialogTheme = {
-  base: 'justify-center items-center flex pointer-events-none top-0 left-0 w-full h-full fixed will-change-transform will-change-opacity',
-  inner:
-    'flex flex-col box-border outline-0 pointer-events-auto overflow-auto max-w-[80vw] max-h-[80vh]',
-  content: 'p-[20px] flex-auto overflow-auto',
-  footer: 'flex p-[20px] pb-[10px]',
-  header: {
-    base: 'flex justify-between pt-[20px] px-[20px] pb-[10px]',
-    text: 'flex-1 m-0 p-0 inline-flex text-3xl font-bold',
-    closeButton:
-      'p-0 m-0 ml-[15px] opacity-80 h-auto w-auto inline-flex bg-none border-none cursor-pointer items-center text-[16px] focus:outline-hidden'
-  }
-};
-
 export const dialogTheme: DialogTheme = {
-  ...baseTheme,
-  inner: [
-    baseTheme.inner,
-    'bg-panel text-text-primary border border-panel-accent rounded-sm shadow-2xl'
-  ].join(' '),
+  base: 'justify-center items-center flex pointer-events-none top-0 left-0 size-full fixed',
+  inner: `
+      flex flex-col box-border outline-0 pointer-events-auto overflow-auto max-w-[80vw] max-h-[80vh] rounded-xl border backdrop-blur-lg
+      border-calendar-colors-container-stroke-default bg-calendar-colors-container-background-default
```

Comment:
use of a calendar color here doesn't seem right - i feel like i had thrown in a couple of incorrect tokens that needed to be updated. this might have been one of them